data/human/transport missions.txt

```diff
@@ -3304,3 +3304,198 @@ mission "Spacediving amateurs"
+				`	"Don't worry, I'm just taking a walk."`
+			label unnerved
+			`The man is starting to act a bit nervous. "Sorry I bothered you. Can we forget this happened?"`
+			choice
+				`	"Well, conversations like this are hard to forget."`
+					goto cornered
+				`	"Nice scam you have running here. Would be a shame if somebody told the local godfather about it."`
+					goto payback
+				`	(Agree.)`
+					goto zerodeal
+			label cornered
+			`	"Well what do you want? Is there anything I can do for you?"`
+			choice
+				`	"No, let's just forget it happened."`
+					goto zerodeal
+				`	"Yes, what an excellent idea. How much do you want to pay me?"`
+			label payback
+			`The man is now really nervous and asks: "How many credits do you need to just forget this whole conversation happened?"`
+			choice
+				`	"100,000,000"`
+					goto far_too_much
+				`	"1,000,000"`
+					goto too_much
+				`	"200,000"`
+					goto too_much
+				`	"50,000"`
+					goto too_much
+				`	"5,000"`
+					goto counterdeal
+				`	"2,000"`
+					goto counterdeal
+				`	"1,000"`
+					goto counterdeal
+				`	"I don't need your credits, just forget it."`
+					goto zerodeal
+			label zerodeal
+			`The man walks away without saying a word.`
+				decline
+			label too_much
+			`	"That's too much! 2389 credits is all I have. Would that be enough?"`
+			choice
+				`	"Sure"`
+					goto counterdeal
+				`	"No way, I want a lot more."`
+					goto far_too_much
+			label counterdeal
+			`	The man hands you a credit chip, nods and walks away. You pocket the chip quickly, more ready to be done with this neighborhood than you want to visibly show.`
+			`A few minutes later you reach into your pocket for the chip... to find it missing.`
+			`	Looks like even a hardened captain like you isn't immune to being pickpocketed on a planet like this.`
+				decline
+			label far_too_much
+			`The man starts panicking and is clumsily reaching in his jacket, maybe to get some money, but more likely reaching for a gun.`
+			choice
+				`	(Try to calm down the man.)`
+					goto draws
+				`	(Mock the man further.)`
+					goto draws
+				`	(Draw your sidearm and shoot the man.)`
+					goto shoot
+			label draws
+			`	The man somehow manages to draw a weapon and tries to shoot at you, but the weapon doesn't seem to function.`
+			choice
+				`	"Ha ha, you are really a failure as a goon."`
+				`	(Draw your sidearm and shoot the man.)`
+					goto shoot
+			`The man tries to fire his weapon once more, but now it does function and the bullet hits you between the eyes. You die almost instantly, but you do have a few microseconds to reflect on the wisdom of mocking a man that is trying to fire a gun at you.`
+				die
+			label shoot
+			`	Your bullet hits the man right between the eyes; he dies instantly, and a crowd starts to form around you.`
+			`Most bystanders are surprised, but you also notice an angry sentiment brewing in the (not unarmed) crowd. It might be wise to leave quickly.`
```

Comment:
```suggestion
			`	Most bystanders are surprised, but you also notice an angry sentiment brewing in the (not unarmed) crowd. It might be wise to leave quickly.`
```